safe_control_gym/envs/gym_pybullet_drones/quadrotor.py

```diff
@@ -274,7 +274,7 @@ def __init__(self,
                 ])  # x = {x, x_dot, y, y_dot, z, z_dot, phi, theta, psi, p_body, q_body, r_body}.
         elif self.TASK == Task.TRAJ_TRACKING:
             POS_REF, VEL_REF, _ = self._generate_trajectory(traj_type=self.TASK_INFO['trajectory_type'],
-                                              traj_length=self.EPISODE_LEN_SEC,
+                                              traj_length=self.EPISODE_LEN_SEC + self.CTRL_TIMESTEP, # Added to extend trajectory by 1 (PR #119)
```

Comment:
This wouldn't happen is the ref is extended in the method it is generated

Reply:
Fixed.